FogCreek/Glitch-Community
# Humdrum triangle: 2FA form for new auth flow

## Links
* Remix link: https://humdrum-triangle.glitch.me/
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/268
* Specs/doc links: https://gallery.io/projects/MCHbtQVoQ2HCZZpaEieUFqhj/files/MCEJu8Y2hyDScckl30Wc2y4V2c6qLxWO8h0

## GIF/Screenshots:

## Changes:
* This changes the 2FA page to match the new designs, and adds a 2FA challenge to the login page

## How To Test:
* Enable "user passwords" and 2FA for your account, then go to https://humdrum-triangle.glitch.me/ and try to log in. You should be required to enter your 2FA code.

## Feedback I'm looking for:

## Things left to do before deploying:
- [ ] I think after we deploy will need to hook this up to the login page after Sarah's username/password is merged.
